CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 - Added libcudadebugger.so to nvliblist.conf to support cuda-gdb in CUDA 12+.
 
+### Bug fixes
+
+- Ensure opened/kept file descriptors in stage 1 are not closed during the Go
+  garbage collection.
```

Comment:
Please explain how this affects the user with something like adding to the end `to avoid "Bad file descriptor" errors at startup`.